src/cheri_insts.sail

```diff
@@ -630,6 +645,11 @@ function clause execute(CRRL(rd, rs1)) =
 }
 
 union clause ast = CRAM : (regidx, regidx)
+/*!
+ * Integer register *rt* is set to a mask that can be used to round addresses
```

Comment:
```suggestion
 * Integer register *rd* is set to a mask that can be used to round addresses
```

Reply:
```suggestion
 * Integer register *rt* is set to the most permissive mask that can be used to round addresses
```
Assuming this is in fact true...

Reply:
I am not sure this is strictly the most permissive mask. I think CRAM right now is precise?

Reply:
Hmm, I'm not sure: it's based on CSetBounds which should be precise. We should probably leave it. Just noticed the asymmetry with CRRL's "smallest value greater or equal".

Reply:
True, smallest in CRRL may also be an overstatement. But if I use "a value" there, then returning UINT64_MAX is a valid implementation.

Reply:
Just as returning 0 here is a valid implementation.

Reply:
UINT64_MAX isn't representable, even if some existing capabilities claim to have that length :)

Reply:
Hmm, that's actually a good point. I recall conversations about this previously, but is CRRL specified to give UINT64_MAX in that case, or 0?